src/main.coffee

```diff
@@ -1,10 +1,12 @@
 matching = require './matching'
 scoring = require './scoring'
 time_estimates = require './time_estimates'
-feedback = require './feedback'
+Feedback = require './feedback'
 
 time = -> (new Date()).getTime()
 
+feedbacker = new Feedback
+
 zxcvbn = (password, user_inputs = []) ->
```

Comment:
Should I add a third parameter to pass the feedback instance (alternatively taking an options object?):

```
const feedbacker = new zxcvbn.Feedback({ warnings: { date: 'foo' }});
const results = zxcvbn('password', [], feedbacker);
```

Or is this acceptable / preferred:
```
const Feedbacker = new zxcvbn.Feedback({ warnings: { date: 'foo' }});
const results = zxcvbn('password', []);
results.feedback = feedbacker.get_feedback(result.score, result.sequence)
```

Or maybe something entirely different?

Reply:
I like the second variant more, it decouples the classes.

Reply:
Okay, then that's the version that's already implemented. I'll guess I'll add a helper method `from_result()` so that one does need to pass two parameters. Are you happy with that name or do you prefer something else? 

It would then look like this:

```
const feedbacker = new zxcvbn.Feedback({ warnings: { date: 'foo' }});
const results = zxcvbn('password', []);
results.feedback = feedbacker.from_result(results)
```

Reply:
It's fine, but I don't have a strong preferance for a specific name either.